int_tests/vectara_int_tests/managers/test_auth.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+from vectara.factory import Factory
+
+
+class TestAuthManager(unittest.TestCase):
+
+    def setUp(self):
+        self.addCleanup(self.cleanup)
+        self.client = Factory().build()
+        response = self.client.app_clients.create(name="test-client", api_roles=["owner"])
+        self.client_id = response.client_id
+        self.client_secret = response.client_secret
+
+    def test_get_access_token(self):
+        response = self.client.auth.get_token(
+            client_id=self.client_id,
+            client_secret=self.client_secret,
+            grant_type="client_credentials"
+        )
+
+        self.assertIsNotNone(response.access_token)
+        self.assertIsNotNone(response.token_type)
+        self.assertIsNotNone(response.expires_in)
+
+    def cleanup(self):
+        for client in self.client.app_clients.list():
+            self.client.app_clients.delete(client.id)
+
+    def tearDown(self):
+        for client in self.client.app_clients.list():
+            self.client.app_clients.delete(client.id)
```

Comment:
Consider calling set.cleanup()

Reply:
Question: does this only test OAuth? If so, why not also add API keys?

Reply:
For the API's we have https://github.com/vectara/python-sdk/pull/27/files#diff-612b6238d5a4eba8b4a51a306f0b6ca8eab05359072a5fbbaae7b711e2c4fe1d